context/context.go

```diff
@@ -63,7 +63,7 @@ func doInit() {
 
 	utils.InitDir(config.LocalStateDir)
 
-	initMyUUID()
+	MyUUID = initMyUUID()
```

Comment:
MyUUID is a global variable in context and the same been referred at `ValidateAdd` in `rpc/services/peer-rpc-svc.go`